src/js/facility-locator/reducers/searchQuery.js

```diff
@@ -27,35 +27,33 @@ export default function (state = INITIAL_STATE, action) {
       return {
         ...state,
         ...action.payload,
+        error: false,
         inProgress: true,
       };
-    case SEARCH_SUCCEEDED:
-      return {
-        ...state,
-        ...action.payload,
-        inProgress: false,
-      };
```

Comment:
I realized this is unneeded because the `FETCH_VA_FACILITIES` and `FETCH_VA_FACILITIES` are the success cases. `SEARCH_SUCCEEDED` is unused. It is a little confusing because there is an intermediate step where the Mapbox API is hit, and needs to return properly before we can fetch the VA facilities